build-parent/pom.xml

```diff
@@ -388,5 +388,18 @@
         <impsort.goal>check</impsort.goal>
       </properties>
     </profile>
+    <profile>
+      <id>quickly</id>
+      <activation>
+        <property>
+          <name>quickly</name>
```

Comment:
Can you please update scripts/MD to mention `-Dquickly`?
See [usages](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/search?q=%22-dskiptests%22)